### ISSUE TITLE
About joint rotation and test set

### ISSUE THREAD

**Issue by jhonsonlaid** [Music-to-dance-motion-synthesis/dataset#2]
Thank you for publishing your dataset. Since  ``skeletons.json``  stores each joint's relative position to the center(root or Hips) and doesn't provide  joint's rotation information explicitly. My problem is:
1. Could you provide original ``.c3d`` format so that I can convert it to ``.bvh`` using Blener or MotionBuilder? Or maybe you already have ``.bvh`` file or some other files that stores joint's rotation?

2. The MM paper didn't mention the test file name, could you tell me how you split the dataset into training set and test set?
